web/src/lib/components/shared-components/side-bar/purchase-info.svelte

```diff
@@ -0,0 +1,175 @@
+      </button>
+    {:else if !$isPurchased && showBuyButton}
+      <button
+        type="button"
+        on:click={openPurchaseModal}
+        on:mouseover={onButtonHover}
+        on:mouseleave={() => (hoverButton = false)}
+        on:focus={onButtonHover}
+        on:blur={() => (hoverButton = false)}
+        class="p-2 flex justify-between place-items-center place-content-center border border-immich-primary/20 dark:border-immich-dark-primary/10 mt-2 rounded-lg shadow-md dark:bg-immich-dark-primary/10 w-full"
+      >
+        <div class="flex justify-between w-full place-items-center place-content-center">
+          <div class="flex place-items-center place-content-center gap-1">
+            <div class="h-6 w-6">
+              <ImmichLogo noText />
+            </div>
+            <p class="flex text-immich-primary dark:text-immich-dark-primary font-medium">
+              {$t('purchase_button_buy_immich')}
+            </p>
+          </div>
+
+          <div>
+            <Icon
+              path={mdiInformationOutline}
+              class="flex text-immich-primary dark:text-immich-dark-primary font-medium"
+              size="18"
+            />
+          </div>
+        </div>
+      </button>
+    {/if}
+  </div>
+{/if}
+
+<Portal target="body">
+  {#if showMessage}
+    <div
+      class="w-[500px] absolute bottom-[75px] left-[255px] bg-gray-50 dark:border-gray-800 border border-gray-200 dark:bg-immich-dark-gray dark:text-white text-black rounded-3xl z-10 shadow-2xl px-8 py-6"
+      transition:fade={{ duration: 150 }}
+      on:mouseover={() => (hoverMessage = true)}
+      on:mouseleave={() => (hoverMessage = false)}
+      on:focus={() => (hoverMessage = true)}
+      on:blur={() => (hoverMessage = false)}
+      role="dialog"
+    >
+      <div class="flex justify-between place-items-center">
+        <div class="h-10 w-10">
+          <ImmichLogo noText />
+        </div>
+        <CircleIconButton
+          icon={mdiClose}
+          on:click={() => {
+            showMessage = false;
+          }}
+          title={$t('close')}
+          size="18"
+          class="text-immich-dark-gray/85 dark:text-immich-gray"
+        />
+      </div>
+
+      <h1 class="text-lg font-medium my-3 dark:text-immich-dark-primary text-immich-primary">
+        {$t('purchase_panel_title')}
+      </h1>
+
+      <div class="text-gray-800 dark:text-white my-4">
+        <p>
+          {$t('purchase_panel_info_1')}
+        </p>
+        <br />
+        <p>
+          {$t('purchase_panel_info_2')}
+        </p>
+      </div>
+      <div class="mt-3 flex flex-col gap-1">
+        <Button fullwidth on:click={openPurchaseModal}>{$t('purchase_button_buy_immich')}</Button>
+        <hr class="my-2" />
+        <Button size="sm" fullwidth on:click={() => hideButton(true)}>{$t('purchase_button_never_show_again')}</Button>
+        <Button size="sm" fullwidth on:click={() => hideButton(false)}>{$t('purchase_button_reminder')}</Button>
```

Comment:
What do you think of this button design? 

![image](https://github.com/user-attachments/assets/e7fb3b32-d47b-41d7-a1d3-2d94d4d5467a)

Reply:
This design is also fine with me

Reply:
Alright, I'll push those changes